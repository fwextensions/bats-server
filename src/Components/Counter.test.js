import React from 'react';
import renderer from 'react-test-renderer';
import Counter from './Counter';

test('Snapshot', () => {
  const component = renderer.create(
    <Counter
      label="Behavioral Beds"
      name="psychBedsCount"
      min={0}
      onChange={() => {}}
      value='val'
    />);

    const componentJson = component.toJSON();
    expect(componentJson).toMatchSnapshot();
});
