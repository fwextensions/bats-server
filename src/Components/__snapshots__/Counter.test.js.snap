// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<div
  className="counter"
>
  <label
    className="counter__label"
    htmlFor="psychBedsCount"
  >
    Behavioral Beds
  </label>
  <div
    className="counter__control"
  >
    <button
      className="usa-button counter__button counter__button--decrement"
      onClick={[Function]}
      type="button"
    >
      âˆ’
    </button>
    <input
      className="usa-input usa-input--small counter__input"
      id="psychBedsCount"
      name="psychBedsCount"
      type="number"
      value="val"
    />
    <button
      className="usa-button counter__button counter__button--increment"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
  </div>
</div>
`;
